MamoPay-Test/Source/Utils + Adapters/NetworkAdapter.swift

```
@@ -0,0 +1,67 @@
+//
+//  NetworkAdapter.swift
+//  MamoPay-Test
+//
+//  Created by Mohammad Imad Ali on 29/05/21.
+//  Copyright © 2021 MamoPay. All rights reserved.
+//
+
+import Foundation
+
+struct WebAPI {
+   static let frequentsURL = "https://60adf30580a61f001733208d.mockapi.io/api/v2/frequents"
+   static let mamoUsersURL = "https://60adf30580a61f001733208d.mockapi.io/api/v2/accounts"
+}
+class NetworkAdapter: NSObject {
+    
+    func apiToGetFrequents(completion: @escaping ([Frequent])->()) {
```

What if you experience an error in the API response here? 
Don't you think using a completion block with error would be beneficial to catch that in the View layer?

---

I have added print statements for now for errors due to time constraints.
Yes, we can return the Error completion handler.